dune-project

```diff
@@ -37,8 +37,7 @@
   cohttp-lwt-unix
   bos
   crunch
-  (mirage-kv-mem
-    (<= 3.1.0))
+  mirage-kv-mem
```

Comment:
Would be good to add a lower bound of `6.0.0` on this I think, just bumped into this because I had an old version and an out of date opam :))

Reply:
I have mixed feelings about this. When I added this, it was sufficient to make `opam` install `mirage-kv.5.0.0`, transitively. Therefore, I didn't add `mirage-kv < 6.0.0`, although it was what was really needed, `mirage-kv` isn't a direct dependency. On the other hand, in a later PR: [#713](https://github.com/ocaml/ocaml.org/pull/713) I added support for `mirage-kv.6.0.0` and removed the constraint.

Overall, I'm not convinced this is done the right way.